client/my-sites/plugins/plugin-details-CTA/usps.tsx

```diff
@@ -133,7 +133,7 @@ export const PlanUSPS: React.FC< Props > = ( {
 	const planDisplayCost = useSelector( ( state ) => {
 		return getProductDisplayCost( state, requiredPlan || '' );
 	} );
-	const monthlyLabel = translate( 'Monthly' );
+	const monthlyLabel = translate( 'month' );
 	const annualLabel = translate( 'Annually' );
```

Comment:
Let's also update this to "year".

![Screenshot 2025-05-14 at 10 18 54 AM](https://github.com/user-attachments/assets/8589d1da-706e-4da6-ae78-adb035a13430)

Reply:
Got it. I was expecting to remove the month/year toggle and was going to pull that out. 👍